api/post_test.go

```diff
@@ -615,7 +613,7 @@ func TestSearchPostsFromUser(t *testing.T) {
 		t.Fatalf("wrong number of posts returned %v", len(result.Order))
 	}
 
-	if result := Client.Must(Client.SearchPosts("from: " + user2.Username + " in:" + channel1.Name)).Data.(*model.PostList); len(result.Order) != 1 {
+	if result := Client.Must(Client.SearchPosts("from: " + user2.Username + " in:" + channel1.Name)).Data.(*model.PostList); len(result.Order) != 0 {
```

Comment:
Ehh there's only two tests that check for zero counts after my changes and nothing they test is not being tested elsewhere. They're also now checking the negative case where nothing should be returned. If you're not 0/5 I'll make the changes but I don't think it's going to give more coverage